src/main/resources/db/changelog/changes/18-create-orders-items-table.yaml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-orders-items-table
+      author: Taras Yurkovskyi
+      changes:
+        - createTable:
+            tableName: orders_items
```

Comment:
`tableName: order_items`

Reply:
why order? in table there are not only items from one order

Reply:
agree with @BohdanTrue :)